### DISCUSSION TITLE
facing issue in build

### DISCUSSION THREAD

**Discussion by garadeaniket** [intel/llvm#11266] [General]
During the installation , below error has been encountered in make command.
cannot find -lLLVMSPIRVLib
is any way to disable this?

**Comment by bader** [comment]
What command do you use to build DPC++ compiler?

**Reply by garadeaniket** [reply]
cmake -G "Unix Makefiles" ../llvm  -DLLVM_ENABLE_PROJECTS='clang' -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=path to /bin/g++   -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_ASSERTIONS=OFF -DLLVM_ENABLE_DUMP=OFF -DLLVM_BUILD_LLVM_DYLIB=ON -DCMAKE_INSTALL_PREFIX=path to install directory -DGCC_INSTALL_PREFIX= path to gcc install directory

**Reply by bader** [reply]
@MrSidims, do you know if LLVM translator requires additional configuration when we build with `-DLLVM_BUILD_LLVM_DYLIB=ON`?

**Reply by MrSidims** [reply]
My expectation would be "no". Meanwhile git history shows, that we had a patch handling the option https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1543 which was later almost completely reverted by the community
